pkg/scheduler/placement/tag_rule_test.go

```diff
@@ -144,3 +144,112 @@ partitions:
 		t.Errorf("tag rule with parent queue incorrect queue '%s', error %v", queue, err)
 	}
 }
+
+// Create the structure for the parent rule tests
+// shared by a number of rule tests
+const confParentChild = `
```

Comment:
I don't see why it has been put to the tag rule. IDEs provide quick jump to the definition, but it's a bit counterintuitive. I think this can be moved to `rule_test.go` or `test_rule.go`, or even a helper go file can be created for this.

Reply:
We cannot move it to `test_rule.go` as that is a part of the production code (i.e. does not end in `_test.go`) and it really does not belong there.
The `rule_test.go` is testing the interface for the rule object, also not really the place to set it as it is not even referenced there. Probably placing it in the top of one of the test files would have been better, just forgot to do that. Adding a new file for just two constants seems a bit much.
There is no real solution that solves it nicely without the need to jump. It might be better for readability to add it to the `rule_test.go` file. It still requires the jumping to find the value, We can do that a a bit of an in between solution, are you OK with that then?

Reply:
Yes, let's keep it here then.